### ISSUE TITLE
Cannot copy stylesheet for project 

### ISSUE THREAD

**Issue by Ilidur** [jonasthewolf/gsn2x#436]
**Describe the bug**

I would like to use the stylesheet functionality without embedding (`-t`). I pass the `-s` flag and the `--output-dir` flag and I get an OS error. If I do the same but with the `-t` the error does not occur and the functionality is present.

**To Reproduce**

```
bin/gsn2x gsn.yaml  --output-dir gsn-svg -s style/status.css
Error: Could not copy stylesheet from /$PROJECT/style/status.css to /$PROJECT/gsn-svg
Caused by:
    Is a directory (os error 21)
```
**Expected behavior**

The SVGs get generated and the stylesheets get copied into the output folder.

**Desktop (please complete the following information):**

 - OS: [Windows under wsl - ubuntu]
 
 - gsn2x Version: gsn2x 3.2.1

**Additional context**

**Comment by jonasthewolf** [comment]
Thanks for your feedback @Ilidur .

You replaced the real path with $PROJECT for the bug report, right?

Does the path contain spaces?

**Comment by Ilidur** [comment]
yes, anonymised project with $PROJECT

No spaces but does contain `-`

**Comment by Ilidur** [comment]
eliminated all `-` but still same result. Added additional output in the description.

**Comment by vjonaswolf** [comment]
The problem here is that the error is from the underlying operating system.
Could you please try `export RUST_BACKTRACE=1` before running the program?
It might provide additional output. If not, we might need a debug build. 
I was actually quite sure that that feature works, so I suspect a WSL issue here. Unfortunately, I don't have access to a WSL.
Have you tried copying the file with `cp` in your terminal?

**Comment by Ilidur** [comment]
Under Windows PS I get the following

```PS
.\bin\gsn2x.exe --stylesheet .\style\status.css .\gsn.yaml --output-dir gsn-svg 
Error: Could not copy stylesheet from \\?\C:\Users\...\docs\design-and-goals\style\status.css to \\?\C:\Users\...\docs\design-and-goals\gsn-svg

Caused by:
    Access is denied. (os error 5)
```

both happily copy (wsl and PS)

**Comment by jonasthewolf** [comment]
The problem here is that the error is from the underlying operating system.
Could you please try export RUST_BACKTRACE=1 before running the program?
It might provide additional output. If not, we might need a debug build.
I was actually quite sure that that feature works, so I suspect a WSL issue here. Unfortunately, I don't have access to a WSL.
Have you tried copying the file with cp in your terminal?

**Comment by Ilidur** [comment]
Backtrace not useful
```
Caused by:
    Is a directory (os error 21)

Stack backtrace:
   0: <unknown>
   1: <unknown>
   2: <unknown>
   3: <unknown>
   4: __libc_start_main
   5: <unknown>
```

I'll try building debug and trying later

**Comment by jonasthewolf** [comment]
The backtrace is actually useful.
Let me check something.

**Comment by jonasthewolf** [comment]
@Ilidur Thanks for the backtraces.

I think I understand the issue now.
I am not appending the file name for the destination.

As described in https://doc.rust-lang.org/std/fs/fn.copy.html, the behaviour of the function I use is platform dependent.

I will have to add the target file name for the call, too.

**Comment by jonasthewolf** [comment]
Let me see if I can fix this earlier than Sunday night.

**Comment by Ilidur** [comment]
No urgency on my end. thanks for the great work!

**Comment by jonasthewolf** [comment]
@Ilidur I just released v3.2.2 which should fix the issue you detected. Please reopen if it did not.
